openedx/core/djangoapps/grading_policy/tests/test_ccx_views.py

```diff
@@ -111,14 +109,34 @@ def setUp(self):
         # Create a course outline
         self.mooc_start = start = datetime.datetime(
             2010, 5, 12, 2, 42, tzinfo=pytz.UTC)
-        chapter = ItemFactory.create(
-            start=start, parent=course, category='sequential')
+        chapter = ItemFactory.create(start=start, parent=course)
         sections = [
             ItemFactory.create(
                 parent=chapter,
                 category="vertical",
-                metadata={'graded': True, 'format': 'Homework'})
-            for _ in xrange(4)]
+                metadata={'graded': True, 'format': 'Homework', 'weight': 0.5}
+            ),
+            ItemFactory.create(
+                parent=chapter,
+                category="vertical",
```

Comment:
please explain new course structure and why we do so and rename sections to vertical